cli/src/main/java/io/github/belgif/rest/guide/validator/cli/BelgifRestGuideCli.java

```diff
@@ -15,7 +15,7 @@
 
 
 @CommandLine.Command(name = "belgif-validate-openapi",
-        description = "The belgif-rest-guide-validator Maven plugin is used to validate if an OpenAPI document conforms to the guidelines in the Belgif REST guide.",
+        description = "The belgif-rest-guide-validator is used to validate if an OpenAPI document conforms to the guidelines in the Belgif REST guide.",
```

Comment:
You can shorten to: "Validate if an OpenAPI document conforms to the guidelines in the Belgif REST guide."